src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -338,7 +338,7 @@ public static ParsedAlbumInfo ParseAlbumTitleWithSearchCriteria(string title, Ar
 
                 simpleTitle = CleanTorrentSuffixRegex.Replace(simpleTitle, string.Empty);
 
-                var releaseRegex = new Regex(@"\b(?<artist>" + artist.Name + @")\b.*\b(?<album>"+ string.Join("|",album.Select(s=>s.Title).ToList()) + @")\b",
+                var releaseRegex = new Regex(@"\b(?<artist>" + Regex.Escape(artist.Name) + @")\b.*\b(?<album>"+ string.Join("|",album.Select(s=>Regex.Escape(s.Title)).ToList()) + @")\b",
```

Comment:
This works to fix problem in most cases, however if artist or album is just a symbol then the word boundary will not work. Can we change this to build the regex string in two parts? Check artist for symbol only then build it, check album for symbol then build it. Then combine strings and build the full regex to check releases.